tests/atomic/main.yml

```diff
@@ -0,0 +1,527 @@
+      when: "'CentOS' in ansible_distribution or ansible_distribution == 'Fedora'"
+
+  roles:
+    - role: atomic_pull
+      apl_image: "{{ cockpit_cname }}"
+
+    - role: atomic_images_list_verify
+      expected_values:
+        repo: "{{ cockpit_cname }}"
+
+    - role: atomic_containers_list_verify
+      expected_values:
+        image_name: "{{ cockpit_cname }}"
+      expect_missing: true
+
+    - role: atomic_install
+      ai_image: "{{ cockpit_cname }}"
+
+    - role: atomic_run
+      ar_image: "{{ cockpit_cname }}"
+
+    - role: atomic_images_list_verify
+      expected_values:
+        repo: "{{ cockpit_cname }}"
+
+    - role: atomic_containers_list_verify
+      expected_values:
+        image_name: "{{ cockpit_cname }}"
+      check_mode: false
+
+    - role: atomic_stop
+      as_container: "{{ aclv_acl_jq_match['id'] }}"
+
+    - role: atomic_containers_delete
+      acd_container: "{{ aclv_acl_jq_match['id'] }}"
+
+    - role: atomic_images_list_verify
+      expected_values:
+        repo: "{{ cockpit_cname }}"
+
+    - role: atomic_containers_list_verify
+      expected_values:
+        image_name: "{{ cockpit_cname }}"
+      expect_missing: true
+
+    - role: atomic_uninstall
+      au_image: "{{ cockpit_cname }}"
+
+    - role: atomic_images_delete
+      aid_image: "{{ ailv_match['id'] }}"
+
+    - role: atomic_images_list_verify
+      expected_values:
+        repo: "{{ cockpit_cname }}"
+      expect_missing: true
+
+    - role: atomic_containers_list_verify
+      expected_values:
+        image_name: "{{ cockpit_cname }}"
+      expect_missing: true
+
+- name: Atomic Short Name
+  hosts: all
+  become: yes
+
+  tags:
+    - shortname
+
+  vars_files:
+    - vars.yml
+
+  pre_tasks:
+    - name: Set cockpit container name for RHELAH
+      set_fact:
+        cockpit_short_name: "rhel7/cockpit-ws"
+        cockpit_fq_name: "registry.access.redhat.com/rhel7/cockpit-ws"
+      when: ansible_distribution == "RedHat"
+
+    - name: Set cockpit container name for Fedora/CentOS
+      set_fact:
+        cockpit_short_name: "cockpit/ws"
+        cockpit_fq_name: "docker.io/cockpit/ws"
+      when: "'CentOS' in ansible_distribution or ansible_distribution == 'Fedora'"
+
+  roles:
+    - role: atomic_pull
+      apl_image: "{{ cockpit_short_name }}"
+
+    - role: atomic_images_list_verify
+      expected_values:
+        repo: "{{ cockpit_fq_name }}"
+
+    - role: atomic_containers_list_verify
+      expected_values:
+        image_name: "{{ cockpit_short_name }}"
+      expect_missing: true
+
+    - role: atomic_install
+      ai_image: "{{ cockpit_short_name }}"
+
+    - role: atomic_run
+      ar_image: "{{ cockpit_short_name }}"
+
+    - role: atomic_images_list_verify
+      expected_values:
+        repo: "{{ cockpit_fq_name }}"
+
+    - role: atomic_containers_list_verify
+      expected_values:
+        image_name: "{{ cockpit_short_name }}"
+      check_mode: false
+
+    - role: atomic_stop
```

Comment:
Might want to drop a comment here to explain the json queries and what you are looking for.